hack/init-buildx.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit -o nounset -o pipefail
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
Is this needed? I didn't see any mention for this in buildx documentation

Reply:
To be honest I copied this from kind project, I suppose it allow you to use buildx at docker version that does not have it as an official feature, let's keep it.